.gitignore

```diff
@@ -1,5 +1,6 @@
 # dependencies
 node_modules/
+**/node_modules
```

Comment:
wondering why node_modules had to be ignored like this. 🤔

Reply:
this was from the seed project, in some cases there may be node_modules that aren't at the top level, doesn't seem to the case in this project.. i could remove